src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -260,3 +260,28 @@ def set_quotas(self, path, max_bytes=None, max_files=None):
         if max_files is not None:
             self.cfs.setxattr(path, 'ceph.quota.max_files',
                               str(max_files).encode(), 0)
+
+    def get_statfs(self, path):
+        """
+        Get the statfs of the specified path by xattr.
+        :param path: The path of the directory/file.
+        :type path: str
+        :return: Returns a dictionary containing 'used_bytes'
+            and 'used_files'.
+        :rtype: dict
+        """
+        logger.debug("get_statfs by xattr of directory: %s", path)
+
+        try:
+            used_bytes = int(self.cfs.getxattr(path, 'ceph.dir.rbytes'))
```

Comment:
this is dashboard right, bytes would be a bit cumbersome to read; i would suggest printing human readable format like KB/MB/GB/TB

Reply:
Can we let the caller convert on their own?

Reply:
I'd recommend displaying in human readable form since it would be much convenient; let's have others opinions here to finalise the code. Although it would not be much of hassle. You would be easily able to code it out in since it's pythonic code and not cpp.

Reply:
Because quota uses byte as return, so I want to consistent with it

Reply:
PR https://github.com/ceph/ceph/pull/49008 allows to set quotas in human readable form; however retrieving quotas is still in bytes. You're right, let's keep this in bytes for now and maybe raise a PR to add it whenever it's need arises.

Reply:
got it